ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/jboss/narayana/lra/main/module.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="org.jboss.narayana.lra">
+
+    <dependencies>
+        <module name="org.jboss.resteasy.resteasy-client"/>
+        <module name="org.jboss.resteasy.resteasy-core-spi"/>
+        <module name="org.jboss.resteasy.resteasy-core"/>
+        <module name="org.jboss.resteasy.resteasy-jaxb-provider"/>
+        <module name="org.jboss.resteasy.resteasy-json-p-provider"/>
+        <module name="jakarta.transaction.api"/>
+        <module name="jakarta.resource.api"/>
+        <module name="jakarta.ws.rs.api" services="export"/>
+        <module name="jakarta.servlet.api"/>
+        <module name="jakarta.xml.bind.api"/>
+        <module name="jakarta.ejb.api"/>
+        <module name="org.jboss.jts"/>
+        <module name="org.jboss.logging"/>
+
+        <!-- TODO WFLY-5966 validate the need for these and remove if not needed.
+             Prior to WFLY-5922 they were exported by javax.ejb.api. -->
+        <module name="javax.rmi.api"/>
+        <module name="org.omg.api"/>
+        <module name="java.xml"/>
+    </dependencies>
+</module>
```

Comment:
Why is this needed?

Reply:
It's unused and it doesn't do anything, as it has no resources of its own and doesn't export any of its dependencies. So it should be removed, along with https://github.com/wildfly/wildfly/pull/18370/files#diff-0a882b46c5e23c165088809789a9549c4737c55d10d01e072fe0fd9666ee439aR309

Reply:
Thank you for your feedback! I created it so that it aligns to the new groupID: org.jboss.narayana.lra
I understand your point and since I will be on PTO from tomorrow I will close this PR so that a new one can be raise from somebody else of the Narayana team while I am away.